src/zope/app/dependable/__init__.py

```diff
@@ -85,12 +85,15 @@ def _make_relative(self, path):
             path = canonicalPath(path)
             if path.startswith(self.pp):
                 path = path[self.pplen:]
-                while path.startswith("/"): # pragma: no cover
-                    # We should not be able to get here. canonicalPath
-                    # doesn't allow trailing / in a path segment, and we
-                    # already cut off the whole length of the parent, which
-                    # we guaranteed to begin and end with a /
-                    path = path[1:]
+                # Now, the path should not actually begin with a /.
+                # canonicalPath doesn't allow trailing / in a path
+                # segment, and we already cut off the whole length of
+                # the parent, which we guaranteed to begin and end
+                # with a /. But it's possible that older dependencies
+                # than we test with could produce this scenario, so we
+                # leave it for BWC.
```

Comment:
BWC = backwards compatibility?  First time I see this acronym.

(We used to use BBB in Zope-land, which doesn't stand for anything, and I guess was just a version of XXX, but for backwards compatibility support code.)